### PR TITLE
[9.0] [port] purchase analytic global

### PR DESCRIPTION
**Pull request by yvaucher**
From 7.0

### CODE REVIEW

File: purchase_analytic_global/views/purchase.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+  <data noupdate="0">
```
Comment (cyrilgdn) [inline purchase_analytic_global/views/purchase.xml]: Can be replaced by `<odoo>` (or at least remove noupdate="0")

**Comment by cyrilgdn** [comment]
:+1:

**Comment by guewen** [comment]
@yvaucher wouldn't it be better to have a computed field with:
- `compute`: if all the lines share the same account, return this account, else it is empty
- `inverse`: write the account in all the lines

This way, the field will never show a value totally unrelated to the lines

**Comment by guewen** [comment]
:+1:

**Review by grindtildeath — APPROVED** [review]
:+1:

File: purchase_analytic_global/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+{'name': 'Purchase - Analytic Account Global',
+ 'version': '9.0.1.0.0',
+ 'author': 'Camptocamp, Odoo Community Association (OCA)',
+ 'maintainer': 'Camptocamp',
+ 'license': 'AGPL-3',
+ 'category': 'Purchase Management',
+ 'complexity': "easy",
+ 'depends': ['purchase',
+             ],
+ 'website': 'http://www.camptocamp.com',
```
Comment (nikul-serpentcs) [inline purchase_analytic_global/__openerp__.py]: https://github.com/OCA/purchase-workflow

File: purchase_analytic_global/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+ 'data': ['views/purchase.xml',
+          ],
+ 'test': [],
```
Comment (nikul-serpentcs) [inline purchase_analytic_global/__openerp__.py]: @yvaucher if not any file then remove key

**Review by nikul-serpentcs — COMMENTED** [review]
netpicking

**Review by i-vyshnevska — APPROVED** [review]

**Comment by yvaucher** [comment]
Travis is broken with :disappointed: 
```
createdb: could not connect to database template1: could not connect to server: No such file or directory

	Is the server running locally and accepting

	connections on Unix domain socket "/var/run/postgresql/.s.PGSQL.5432"?
```

**Pull request merged**